docs/specification.md

```diff
@@ -292,9 +292,9 @@ The "Merge Contributors Modal" includes:
 
 The "Profile" page includes:
 
```

Comment:
What about the ability to edit the profile picture?

Reply:
I fixed in [commit](https://github.com/BinaryStudioAcademy/bsa-2024-gitfit/pull/98/commits/ae3b1428b1e5012672347a3ebf30b04d9e38c4e7)